plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -421,6 +422,7 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
       fc::microseconds                _ro_read_window_time_us{ 60000 };
       static constexpr fc::microseconds _ro_read_window_minimum_time_us{ 10000 };
       fc::microseconds                _ro_read_window_effective_time_us{ 0 }; // calculated during option initialization
+      fc::time_point                  _read_window_deadline; // only modified on app thread
```

Comment:
Can this be named as` _ro_read_window_deadline` to be consistent with other names?

Reply:
Renamed